lib/exabgp/reactor/listener.py

```diff
@@ -52,19 +52,21 @@ def listen (self, local_ip, peer_ip, local_port, md5, md5_base64, ttl_in):
 				continue
 			if local_port != port:
 				continue
-			if md5:
-				MD5(sock,peer_ip.top(),0,md5,md5_base64)
+			MD5(sock,peer_ip.top(),0,md5,md5_base64)
```

Comment:
Hi @brijohn - why was the if test removed here ? accident ? it work without it ?